apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -24,6 +25,90 @@ namespace Google.Cloud.Bigtable.V2
+        /// is not currently available to most Cloud Bigtable customers. This feature
+        /// might be changed in backward-incompatible ways and is not recommended for
+        /// production use. It is not subject to any SLA or deprecation policy.
+        ///
+        /// This value specifies routing for replication. If not specified, the
+        /// "default" application profile will be used.
+        /// </param>
+        /// <returns>The task representing the created <see cref="BigtableClient"/>.</returns>
+        public static async Task<BigtableClient> CreateAsync(ServiceEndpoint endpoint = null, BigtableServiceApiSettings settings = null, string appProfileId = null)
+        {
+            var client = await BigtableServiceApiClient.CreateAsync(endpoint, settings).ConfigureAwait(false);
+            return new BigtableClientImpl(client, appProfileId);
+        }
+
+        /// <summary>
+        /// Synchronously creates a <see cref="BigtableClient"/>, applying defaults for all unspecified settings,
+        /// and creating a channel connecting to the given endpoint with application default credentials where
+        /// necessary.
+        /// </summary>
+        /// <param name="endpoint">Optional <see cref="ServiceEndpoint"/>.</param>
+        /// <param name="settings">Optional <see cref="BigtableServiceApiSettings"/>.</param>
+        /// <param name="appProfileId">
+        /// This is a private alpha release of Cloud Bigtable replication. This feature
+        /// is not currently available to most Cloud Bigtable customers. This feature
+        /// might be changed in backward-incompatible ways and is not recommended for
+        /// production use. It is not subject to any SLA or deprecation policy.
+        ///
+        /// This value specifies routing for replication. If not specified, the
+        /// "default" application profile will be used.
+        /// </param>
+        /// <returns>The created <see cref="BigtableClient"/>.</returns>
+        public static BigtableClient Create(ServiceEndpoint endpoint = null, BigtableServiceApiSettings settings = null, string appProfileId = null)
+        {
+            var client = BigtableServiceApiClient.Create(endpoint, settings);
+            return new BigtableClientImpl(client, appProfileId);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="BigtableClient"/> which uses the specified channel for remote operations.
+        /// </summary>
+        /// <param name="channel">The <see cref="Channel"/> for remote operations. Must not be null.</param>
+        /// <param name="settings">Optional <see cref="BigtableServiceApiSettings"/>.</param>
+        /// <param name="appProfileId">
+        /// This is a private alpha release of Cloud Bigtable replication. This feature
+        /// is not currently available to most Cloud Bigtable customers. This feature
+        /// might be changed in backward-incompatible ways and is not recommended for
```

Comment:
I'm nervous about this statement - we should talk about what sort of backward-incompatibility we mean here. Might the actual signatures change, or is it just the semantics?

Reply:
This is copied from the `AppProfileId` property docs on one of the request classes. I figured since this library was also alpha, it would be ok to add this here and potentially break it, but if you'd feel more comfortable, we could add overloads or new methods entirely which allow you to specify this parameter.

Reply:
Maybe leave it as it is, but add an issue so that we don't forget to address this (one way or the other) before the library goes GA?

Reply:
Oh, and to answer your question, I can't say if the signature will change. I can ask internally if you'd like.

Reply:
Good idea. Will do.